Index: public_html/js/inscription.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/public_html/js/inscription.js b/public_html/js/inscription.js
new file mode 100644
--- /dev/null	(date 1655284454088)
+++ b/public_html/js/inscription.js	(date 1655284454088)
@@ -0,0 +1,37 @@
+function display_inscription(){
+    // Display
+    document.getElementById("notification").classList.add("d-none");
+    document.getElementById("user_name").classList.add("d-none");
+    document.getElementById("setting").classList.add("d-none");
+
+    // Creation content
+    let content = document.getElementById("content");
+    let form_inscription = document.createElement("div");
+    form_inscription.id = "form_inscription";
+    form_inscription.classList.add("col-md-4");
+    form_inscription.innerHTML = "" +
+        "<form class='form'>" +
+        "   <div class='input-group'>\n" +
+        "       <label for='mail' class='input-group-text'>Email ></label>\n" +
+        "       <input class='form-control' type='text' id='mail' placeholder='Entrez votre email'/>\n" +
+        "   </div>" +
+        "   <div class='input-group'>\n" +
+        "       <label for='pwd' class='input-group-text'>Mot de passe ></label>\n" +
+        "       <input class='form-control' type='password' id='pwd' placeholder='Entrez votre mot de passe'/>\n" +
+        "   </div>" +
+        "   <button type='submit' class='btn'>\n" +
+        "       Se connecter\n" +
+        "   </button>\n" +
+        "</form>\n" +
+        "<button type='' class='btn'>S'inscrire</button>";
+    content.innerHTML = "" +
+        "<div class='row'>" +
+        "   <span class='col-md-4'></span>" +
+        form_inscription.outerHTML +
+        "</div>";
+}
+function listener_inscription() {
+
+}
+//test
+display_inscription();
\ No newline at end of file
